sklearn/linear_model/logistic.py

```diff
@@ -1705,10 +1731,12 @@ class LogisticRegressionCV(LogisticRegression, BaseEstimator,
         l2 penalty with liblinear solver. Prefer dual=False when
         n_samples > n_features.
 
-    penalty : str, 'l1', 'l2', or 'elasticnet', optional (default='l2')
+    penalty : str, 'l1', 'l2', 'elasticnet' or 'none', optional (default='l2')
```

Comment:
What's the point of supporting none in CV, when its role is to determine the optimal C under cross-validation?

Reply:
Good point, I was in auto pilot ^^